Project.toml

```diff
@@ -67,4 +70,5 @@ UnsafeAtomicsLLVM = "0.1"
 julia = "1.8"
 
 [extras]
+ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
```

Comment:
I'm not sure if this is necessary, but I saw it was done for SpecialFunctions, so I added it here for consistency.